redfish.conf

```diff
@@ -13,7 +13,7 @@ userName = administrator
 password =
 
 [ssl]
-SSLType = adhoc
+SSLType = disabled
```

Comment:
Did you mean to change this? I prefer disabled.

Reply:
I prefer too... but not on this patch! It's not NetworkFunction related.

Reply:
Yes I meant to. It's really a pain have to accept the cert every time during development. Ok, we change it in another patch